src/app/reporting/Engine.cpp

```diff
@@ -336,6 +377,78 @@ void Engine::Run()
         mCurReadHandlerIdx = (mCurReadHandlerIdx + 1) % CHIP_IM_MAX_NUM_READ_HANDLER;
         readHandler        = imEngine->mReadHandlers + mCurReadHandlerIdx;
     }
+    for (auto & path : mDirtyPathList)
+    {
+        path.ClearDirty();
```

Comment:
this makes mDirtyPathList NOT have the dirty flag anymore, so the name is misleading. Please rename to something that represents what it contains (it is not the 'list of dirty paths' because here they are obviously not dirty anymore)

Reply:
yep, rename it to mChangeSetPathList, thanks